code/datums/black_market_sellable.dm

```diff
@@ -143,4 +143,40 @@ var/list/black_market_sellables = list()
 	return get_price()
 */
 
+/datum/black_market_sellable/xenoarch
+	name = "Misc Xeno Artifacts"
+	category = "Misc Muses and Artifacts"
+	demand_min = -1
+	demand_max = -1
+	price_min = 50
+	price_max = 120
+	display_chance = 100
+	item = /obj/item
```

Comment:
will this conflict with other sellables?

Reply:
Yes it will, actually. It's a for loop of each black_market_sellable, but it returns 1 or 0 if the purchase isn't valid but it matches the type

Reply:
Sorted